tests/test_fib.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+from netaddr import *
+import time
+import logging
+import requests
+from ptf_runner import ptf_runner
+from ansible_host import AnsibleHost
+from datetime import datetime 
+
+@pytest.mark.parametrize("ipv4, ipv6, mtu", [pytest.param(True, True, 1514)])
+def test_fib(ansible_adhoc, testbed, duthost, ptfhost,ipv4, ipv6, mtu):
+
+    t1_topologies =  ['t1', 't1-lag', 't1-64-lag', 't1-64-lag-clet']
+    t0_topologies =  ['t0',  't0-52', 't0-56', 't0-64', 't0-64-32', 't0-116']
```

Comment:
I am wondering if you could make this more generic?

Having these 2 above list meaning that if we define a new topology in the future. This test code needs to be changed too. Previously we only have to change the testcase.yml.

Reply:
good concern, this needs to work together with the new fib fixture. suggest to address in a separate pr.